pkg/schematic/schematic.go

```diff
@@ -46,6 +48,13 @@ type SystemExtensions struct {
 	OfficialExtensions []string `yaml:"officialExtensions,omitempty"`
 }
 
+// Overlay represents the overlay options for image generation.
+type Overlay struct {
+	Options map[string]any `yaml:"options,omitempty"`
```

Comment:
I think we should reverse the order here: `image`, `name` and `options`

Reply:
that was to make govet happy :sweat_smile:

Reply:
Nolint it!